lib/jquery/assert_select.rb

```diff
@@ -91,14 +91,14 @@ def assert_select_jquery(*args, &block)
 
     if block_given?
       @selected ||= nil
-      fragments = Nokogiri::HTML::Document.new
+      fragments = Nokogiri::HTML::Document.new.fragment
```

Comment:
If preferred this over `Nokogiri::HTML::DocumentFragment.new` because I woudl have been forced to pass an already existing HTML Document. And also preferred this over `Nokogiri::HTML::DocumentFragment.parse` to avoid the empty string parameter that would obscure the intention of this line of code.